src/transactions/TransactionFrame.h

```diff
@@ -55,11 +55,13 @@ class TransactionFrame
 
     enum ValidationType
     {
-        kInvalid,             // transaction is not valid at all
+        kInvalid,                     // transaction is not valid at all
+        kInvalidRemoveOneTimeSigners, // transaction is invalid but one-time
+                                      // signers should be removed
         kInvalidUpdateSeqNum, // transaction is invalid but its sequence number
                               // should be updated
         kInvalidPostAuth,     // transaction is invalid but its sequence number
-                              // should be updated and one-time signers removed
+                              // should be updated
```

Comment:
A few nitpicks related to `ValidationType` and the corresponding comments:
1. Let's rename `kInvalidPostAuth` to something like `kMaybeValidCheckSignatures` because that is what the result actually means
2. Let's improve the comments for `kInvalidUpdateSeqNum` and `kInvalidPostAuth` to better reflect what they mean:
- `kInvalidUpdateSeqNum` means to update the sequence number and remove one-time signers
- `kInvalidPostAuth` means to update the sequence number, check signatures, and remove one-time signers

This should make this code much easier to maintain in the future.

Reply:
see my other comment, I think this is moot now